docs/source/usage.rst

```diff
@@ -396,14 +396,12 @@ Custom interpolations
 
 You can add additional interpolation types using custom resolvers.
 This example creates a resolver that adds 10 to the given value
-(note the need to specify `args_as_strings=False` when the resolver takes non-string arguments --
-this is to preserve backward compatibility with the old behavior where resolver arguments were always given
-as strings).
+(note that the resolver's first argument is ignored here, we will
+see below how it can be used to access the config object itself).
```

Comment:
```suggestion
The first argument of the resolver is the node containing the interpolation, it's not used in this example.
Later examples will demonstrate how to use it.
```

Reply:
I used your suggestion -- up to some minor modifications -- in dd29ad4